modules/govuk_jenkins/manifests/jobs/integration_deploy.pp

```diff
@@ -4,9 +4,6 @@
 #
```

Comment:
Am I right in thinking we only still need this to deploy things like puppet and licensify?